news/4645-use-conda-index

```diff
@@ -0,0 +1,22 @@
+### Enhancements
+
+* Depend on standalone conda-index instead of bundled indexing code. (#4690)
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* `conda-build index --file` is no longer supported.
+* `conda-build index --hotfix-source-repo` is removed.
```

Comment:
Now we anticipate removing the entire `conda index` instead of removing the two features not implemented in standalone `conda-index`